src/model.ts

```diff
@@ -1,69 +1,152 @@
 // Review id => Review
 export const REVIEWS: Map<string, Review> = new Map();
-// Project Paper id => Ids of Reviews
+// Project Paper id => Ids of reviews for this project paper
 export const PAPER_REVIEWS: Map<string, string[]> = new Map();
 // Paper Id => PDF Blob
 export const PAPER_PDFS: Map<string, Uint8Array> = new Map();
 
-export function addProjectPaperReviews(paper: ServerProjectPaper): Project_Paper {
-    return {
-        ...paper,
-        reviews: PAPER_REVIEWS.get(paper.id)!.map((reviewId) => REVIEWS.get(reviewId)!),
-    };
+export interface ExampleData {
+    criteria?: Criterion[];
+    papers?: Paper[];
+    users?: User[];
+    userSettings?: UserSettings[];
+    projects?: Project[];
+    projectMembers?: { projectId: string; members: Project_Member[] }[];
+    reviews?: Review[];
+    projectPapers?: Project_Paper[];
 }
 
-function isEnabled(option: string | undefined): boolean {
-    option = option?.toLowerCase() ?? "";
-    return option == "1" || option == "yes" || option == "true";
-}
+/**
+ * Processes the loaded example data, i.e. (eventually) initializing the maps (= "database")
+ * containing all the data of the mock backend.
+ *
+ * @param data the loaded example data
+ */
+function processExampleData(data: ExampleData) {
+    data.criteria?.forEach((criterion) => CRITERIA.set(criterion.id, criterion));
+    data.papers?.forEach((paper) => PAPERS.set(paper.id, paper));
+    data.reviews?.forEach((review) => REVIEWS.set(review.id, review));
+    data.users?.forEach((user) => {
+        USERS.set(
+            user.email,
+            fromUser(user, `user${user.id}`, { accessToken: "", refreshToken: "" }),
+        );
+
+        // create random reading list for this user
+        READING_LISTS.set(user.email, getRandomItems(Array.from(PAPERS.values()), 4, 10));
 
-if (isEnabled(process.env.ENABLE_DUMMY_ADMIN)) {
-    LOG.warn("Security Risk: Dummy Admin User Enabled!");
-    USERS.set("admin@admin", {
-        id: "admin@admin",
-        email: "admin@admin",
-        firstName: "admin",
-        lastName: "admin",
-        role: UserRole.ADMIN,
-        status: UserStatus.ACTIVE,
-        password: "admin",
-        accessToken: "admin",
-        refreshToken: "admin",
+        // add user settings
+        USER_SETTINGS.set(user.email, getRandomItems(data.userSettings ?? [])[0]);
     });
-    LOG.info(USERS.get("admin@admin"), "The dummy admin user");
+    data.projects?.forEach((project) => {
+        PROJECTS.set(project.id, project);
+        PROJECT_PROJECT_PAPERS.set(project.id, []);
+
+        // create project criteria
+        PROJECT_CRITERIA.set(project.id, getRandomItems(Array.from(CRITERIA.keys()), 5, 10).sort());
+
+        // create a set of project papers
+        getRandomItems(Array.from(data.projectPapers ?? []), 25, 40)
+            .filter((paper) => paper.stage <= project.maxStage)
+            .map((paper) => ({ ...paper, id: project.id + "-" + paper.id }))
```

Comment:
Use string interpolation ([`typescript:S3512`](https://rules.sonarsource.com/typescript/impact/maintainability/RSPEC-3512/))
```suggestion
            .map((paper) => ({ ...paper, id: `${project.id}-${paper.id}` }))
```